modular_skyrat/code/modules/mining/equipment/regenerative_core.dm

```diff
@@ -29,6 +29,11 @@
 	if(inert)
 		to_chat(user, "<span class='notice'>[src] has decayed and can no longer be used to heal.</span>")
 		return
+	if(AmBloodsucker(H))
+		to_chat(user, "<span class='notice'>[src] shrivels up and turns into dust as it touches [H].</span>")
+		new /obj/effect/decal/cleanable/ash(get_turf(H))
```

Comment:
Wont this allow bloodsuckers to be meta checked with 100 percent certainty? By other people applying cores